src/libraries/System.Net.Sockets/tests/FunctionalTests/System.Net.Sockets.Tests.csproj

```diff
@@ -57,45 +58,26 @@
     <Compile Include="UdpClientTest.cs" />
     <Compile Include="UnixDomainSocketTest.cs" />
     <!-- Common Sockets files -->
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.cs"
-             Link="SocketCommon\Configuration.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.Sockets.cs"
-             Link="SocketCommon\Configuration.Sockets.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\TestSettings.cs"
-             Link="SocketCommon\TestSettings.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\Fletcher32.cs"
-             Link="SocketCommon\Fletcher32.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestExtensions.cs"
-             Link="SocketCommon\SocketTestExtensions.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestServer.cs"
-             Link="SocketCommon\SocketTestServer.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestServerAsync.cs"
-             Link="SocketCommon\SocketTestServerAsync.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestServerAPM.cs"
-             Link="SocketCommon\SocketTestServerAPM.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketImplementationType.cs"
-             Link="SocketCommon\SocketImplementationType.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Configuration.cs" Link="SocketCommon\Configuration.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Configuration.Sockets.cs" Link="SocketCommon\Configuration.Sockets.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\TestSettings.cs" Link="SocketCommon\TestSettings.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\Fletcher32.cs" Link="SocketCommon\Fletcher32.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestExtensions.cs" Link="SocketCommon\SocketTestExtensions.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestServer.cs" Link="SocketCommon\SocketTestServer.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestServerAsync.cs" Link="SocketCommon\SocketTestServerAsync.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketTestServerAPM.cs" Link="SocketCommon\SocketTestServerAPM.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\Sockets\SocketImplementationType.cs" Link="SocketCommon\SocketImplementationType.cs" />
```

Comment:
Visual Studio keeps reformatting of the whole `.csproj` when you make a small change. All changes could be reverted now.

Reply:
fixed, nice catch. sry for missing that